device-types/Cisco/C9200L-24PXG-2Y.yaml

```diff
@@ -0,0 +1,128 @@
+---
+manufacturer: Cisco
+model: C9200L-24PXG-2Y
+part_number: C9200L-24PXG-2Y
+slug: cisco-c9200l-24pxg-2y
+u_height: 1
+is_full_depth: false
+comments: '[Cisco Catalyst 9200 Series Switches Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/catalyst-9200-series-switches/nb-06-cat9200-ser-data-sheet-cte-en.html)'
+console-ports:
+  - name: Console
+    type: rj-45
+  - name: usb
+    type: usb-mini-b
+power-ports:
+  - name: PS-1
+    type: iec-60320-c14
+    maximum_draw: 600
+  - name: PS-2
+    type: iec-60320-c14
+    maximum_draw: 600
```

Comment:
These PSU's are modular and thus you should define `module-bays` with 2 slots. Then, the applicable PSU modules should be added to the module-types folder if they do not already exist

Reply:
Thanks! Added module bays (see other reply)